robottelo/cli/factory.py

```diff
@@ -3858,13 +3871,20 @@ def setup_cdn_and_custom_repos_content(
     if missing_subscription_names:
         raise CLIFactoryError(
             u'Missing subscriptions: {0}'.format(missing_subscription_names))
-
-    return dict(
+    data = dict(
         activation_key=activation_key,
         content_view=content_view,
         product=custom_product,
         repos=repos_info,
     )
+    if lce_id:
+        lce = LifecycleEnvironment.info({
+            'id': lce_id,
+            'organization-id': org_id,
+        })
+        data['lce'] = lce
```

Comment:
This helper function is receiving lce_id as argument, why we need to returning back?

Reply:
To have everything related to our repo collection stored in one place so it can be easily reused later?